files/slave.dockerfile

```diff
@@ -25,13 +24,24 @@ RUN yum install -y python-pip python-devel && \
     yum groupinstall -y 'Development Tools' && \
     # Very specific version is required for older Openstack Env.
     pip install --no-cache-dir --disable-pip-version-check shade==1.12.1 && \
+
+    # Use Ansible from source until rpm available w/ fix for:
+    # https://github.com/ansible/ansible/issues/15915
+    pip install --upgrade setuptools && \
```

Comment:
The ``python-setuptools`` installed from  ``slave.rpms`` is too old, temporarily bump it up using pip (eek!). Effectively, the install in ``slave.rpms`` is just to make sure as many dependencies come in rpm-format as possible (so pip ignores them).